docs/upgrade.md

```diff
@@ -18,16 +18,34 @@ $ brew update && brew upgrade tilt-dev/tap/tilt
 $ brew install tilt-dev/tap/tilt
 ```
 
-### If you installed with the install script or directly with a release binary
+### If you installed with the install script
```

Comment:
note that the install script uses homebrew, so it feels weird to put homebrew above the install script in the list of options

can we reorganize this to be more like the install page?

https://docs.tilt.dev/install.html

i.e., point people to the installer script as the primary upgrade path, and give them alternative options at the bottom?

note that this is also missing the windows install script (which is another point in favor of reorganizing, since we seem to be missing things)

Reply:
Yeah I like that idea, will do

Reply:
@nicks reorganized! LMK what you think